### ISSUE TITLE
overfull vbox and hbox

### ISSUE THREAD

**Issue by pfetsch** [tudace/tuda_latex_templates#36]
When I compile ``DEMO-TUDaBeamer.tex``, I get overful ``vbox`` and ``hbox``es. For example
```
Overfull \vbox (4.69463pt too high) detected at line 41
Overfull \hbox (20.0pt too wide) has occurred while \output is active
```
I seems to me that they are produced by the style, not the particular example.

**Comment by TeXhackse** [comment]
currently working on that

**Comment by pfetsch** [comment]
Very good - thank you.

**Comment by TeXhackse** [comment]
resolved by 150145f3aa4af9a39e96a513aca1109ffccd9cf5 and 382ff5ad492fb8b732db4a208dc9cd7e265ea09f
